assignment.js

```diff
@@ -32,6 +32,11 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+let baseNumber = 9
```

Comment:
🛠 Read the comment below and update your code accordingly

@jimernest90 while your approach is solid, there are a few issues that need to be cleaned up:

1. For readability purposes, your `for` loop should be structured like this:

```
for (let i = 1; i < 11, i++) {
 //logic goes here
}
```

2. You are missing the variable declaration for your iterator `i` inside the parentheses of the `for` loop

It should be `let i = 1` and not `i=1`

3. The assignment called for the program to print out multiples up to 12, your program stops at 11